Commands.cs

```diff
@@ -19,6 +21,127 @@ public Commands(WarpplateManager manager, Player[] players)
             Players = players;
         }
 
+        public void MainCommand(CommandArgs args)
+        {
+            string cmd = "help";
+            if (args.Parameters.Count > 0)
+            {
+                cmd = args.Parameters[0].ToLower();
+            }
+            switch (cmd)
+            {
+                case "set":
+                    SetWarpplate(args);
+                    break;
+
+                case "del":
+                case "delete":
+                case "remove":
+                    DeleteWarpplate(args);
+                    break;
+
+                case "setdest":
+                case "setdestination":
+                    SetWarpplateDestination(args);
+                    break;
+
+                case "deldest":
+                case "deletedestination":
+                    RemoveWarpplateDestination(args);
+                    break;
+
+                case "info":
+                    WarpplateInformation(args);
+                    break;
+
+                case "list":
+                    ListWarpplates(args);
+                    break;
+
+                case "allow":
+                case "toggle":
+                    WarpplateAllow(args);
+                    break;
+
+                case "reload":
+                    ReloadWarpplates(args);
+                    break;
+
+                case "delay":
+                    SetWarpplateDelay(args);
+                    break;
+
+                case "width":
+                    SetWarpplateWidth(args);
+                    break;
+
+                case "height":
+                    SetWarpplateHeight(args);
+                    break;
+
+                case "resize":
+                case "size":
+                    SetWarpplateSize(args);
+                    break;
+
+                case "label":
+                case "display":
+                    SetWarpplateLabel(args);
+                    break;
+
+                case "dim":
+                case "dimension":
+                    if (!args.Player.HasPermission("warpplate.setdimensional"))
+                    {
+                        args.Player.SendErrorMessage("You do not have permission to set dimensions.");
```

Comment:
Can this be "You do not have permission to set a dimensional warpplate." ?

Reply:
Resolved